sdk/packages/eternum/src/utils/battleSimulation.ts

```
@@ -1,24 +1,34 @@
 
-class Percentage {
+export class Percentage {
   static _100() {
     return 10_000;
   }
 
-  static get(value, numerator) {
+  static get(value: number, numerator: number) {
     return (value * numerator) / Percentage._100();
   }
 }
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)